.tito/cockpit-build.sh

```diff
@@ -8,13 +8,8 @@ if [ -n "${USE_SUSE}" ] ; then
 fi
 
 echo "Generating cockpit dist-gzip..."
-if [ ! $(which yarn) ] ; then
-  echo "Unable to build cockpit, yarn not installed (see README)"
-  exit 1
-fi
 pushd cockpit
-yarn cache clean
-yarn install --frozen-lockfile
+npm ci
```

Comment:
`npm ci` info > https://docs.npmjs.com/cli/ci.html
for more reproducible builds from the lockfile
we could also change this to `npm ci --cache $(mktemp -d)`  if we want to force re-download all the deps